recommenders/popularity_recommender.py

```diff
@@ -1,31 +1,18 @@
-from recommenders import data_movies, data_ratings, default_poster_url
-import pandas as pd
-
-
-def recommend_popular_movies():
-
```

Comment:
The variable name 'movies_ratings_merged' should be separated by underscores for better readability.